hknweb/candidate/models.py

```diff
@@ -56,35 +55,25 @@ def rejected(self):
     def __str__(self):
         return self.name
 
-class CandidateForm(models.Model):
-
-    name           = models.CharField(max_length=MAX_STRLEN, default='')
-    duedate        = models.DateTimeField(default=timezone.now)
-    link           = models.CharField(max_length=MAX_STRLEN, default='')
-    # if visible == False, then admins can see candiate form but it's not displayed on portal
-    visible         = models.BooleanField(default=False)
-
-    def __str__(self):
-        return self.name
-
 
 class BitByteActivity(models.Model):
```

Comment:
Since there are two types of BitByte activities (boba and adventure), it may be helpful to VP to separate these out to more accurately check whether certain candidates have satisfied their requirements. Definitely a minor point since most BitByte families are responsible in doing both the boba and adventure activities and because we're usually very lenient about things like this, so this is in pursuit of matching our requirements exactly.

Reply:
Should have maybe brought this up in the last review, but didn't realize it, apologies!

Reply:
Danny said it was fine